recipes/read-excel/1.1.4.1/test_package/conanfile.py

```diff
@@ -0,0 +1,15 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class ReadExcelTestConan(ConanFile):
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "read-excel.test")
+            self.run(bin_path, run_environment=True)
```

Comment:
To avoid a too strong connection between the test_package directory layout and the way cmake/conan builds, it might be better to pass the test file as an argument
```suggestion
            bin_path = os.path.join("bin", "read-excel.test")
            xls_path = os.path.join(self.source_folder, "sample.xls");
            self.run("{} \"{}\"", run_environment=True)
```